spec/models/article_search_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Article do
   describe ".text_search" do
-    before { 100.times { create :article } }
+    before { 10.times { create :article } }
```

Comment:
@olivierlacan I'm not sure if setting the amount of records to be created to an arbitrarily high number helps tease out weird failures. If anything I would like to move that to a at least a `before(:all)` so it only has to be done once. Right now it is creating around 600 records for the six or seven specs. 

Also not sure what kind of machine you're running specs on but it is not shaving 'a few seconds' off. I've run the suite on a few machines ( that are newer and maxed out ) and am getting similar results on time.

__Couple year old iMac__
7 min 1 sec using `before` with 100 records created
1 min 37 sec using `before(:all)` with 100 records created

__Maxed out year old Macbook Pro__
4 min 9 sec using `before` with 100 records created
1 min 10 using `before(:all)` with 100 records created

I'm wondering if you have a specific case where having more records was able to help you find an error?